### PR TITLE
Fixed metadata docs

### PR DESCRIPTION
**Pull request by htahir1**
## Describe changes
Fixed some things which were false. Once merged please help me out with a backport!

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).
- [ ] If my changes require changes to the dashboard, these changes are communicated/requested.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: docs/book/stacks-and-components/component-guide/experiment-trackers/wandb.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ from zenml.client import Client
 
 last_run = client.get_pipeline("<PIPELINE_NAME>").last_run
 trainer_step = last_run.get_step("<STEP_NAME>")
-tracking_url = trainer_step.run_metadata.get("experiment_tracker_url")
+tracking_url = trainer_step.run_metadata["experiment_tracker_url"]
```
Comment (strickvl) [inline docs/book/stacks-and-components/component-guide/experiment-trackers/wandb.md:188]:
```suggestion
tracking_url = trainer_step.run_metadata["experiment_tracker_url"].value
```

**Review by strickvl — CHANGES_REQUESTED** [review]

**Review by strickvl — APPROVED** [review]

File: docs/book/stacks-and-components/component-guide/experiment-trackers/neptune.md [hunk 2]
```diff
@@ -184,7 +184,7 @@ from zenml.client import Client
 
 last_run = client.get_pipeline("<PIPELINE_NAME>").last_run
 trainer_step = last_run.get_step("<STEP_NAME>")
-tracking_url = trainer_step.run_metadata.get("experiment_tracker_url")
+tracking_url = trainer_step.run_metadata["experiment_tracker_url"].value
 print(tracking_url.value)
```
Comment (avishniakov) [inline docs/book/stacks-and-components/component-guide/experiment-trackers/neptune.md:188]: This code was valid - we do `print(tracking_url.value)` below and `.get` is a valid dict operation.

File: docs/book/stacks-and-components/component-guide/experiment-trackers/mlflow.md [hunk 3]
```diff
@@ -224,7 +224,7 @@ from zenml.client import Client
 
 last_run = client.get_pipeline("<PIPELINE_NAME>").last_run
 trainer_step = last_run.get_step("<STEP_NAME>")
-tracking_url = trainer_step.run_metadata.get("experiment_tracker_url")
+tracking_url = trainer_step.run_metadata["experiment_tracker_url"].value
 print(tracking_url.value)
```
Comment (avishniakov) [inline docs/book/stacks-and-components/component-guide/experiment-trackers/mlflow.md:228]: This code was valid - we do `print(tracking_url.value)` below and `.get` is a valid dict operation.

File: docs/book/stacks-and-components/component-guide/experiment-trackers/wandb.md [hunk 4]
```diff
@@ -185,7 +185,7 @@ from zenml.client import Client
 
 last_run = client.get_pipeline("<PIPELINE_NAME>").last_run
 trainer_step = last_run.get_step("<STEP_NAME>")
-tracking_url = trainer_step.run_metadata.get("experiment_tracker_url")
+tracking_url = trainer_step.run_metadata["experiment_tracker_url"].value
 print(tracking_url.value)
```
Comment (avishniakov) [inline docs/book/stacks-and-components/component-guide/experiment-trackers/wandb.md:189]: This code was valid - we do `print(tracking_url.value)` below and `.get` is a valid dict operation.

**Review by avishniakov — CHANGES_REQUESTED** [review]
I would leave Experiment Tracker codes as is or rework further. Rest looks good

**Comment by htahir1** [comment]
@avishniakov Thanks for the catch. I updated and kept the syntax for conformity

**Comment by avishniakov** [comment]
> @avishniakov Thanks for the catch. I updated and kept the syntax for conformity

There is no commit with changes.

**Comment by htahir1** [comment]
@avishniakov sorry now

**Review by avishniakov — APPROVED** [review]

**Pull request merged**